src/main.go

```diff
@@ -3,20 +3,123 @@ package main
 import (
 	"fmt"
 	"game"
+	"time"
 	"os"
 	"os/exec"
+	"os/signal"
 	"renderer"
+	"syscall"
+	"github.com/nsf/termbox-go"
+	
 )
 
 func clearTerminal() {
-	c := exec.Command("clear")
-	c.Stdout = os.Stdout
-	c.Run()
+	clearCmd := exec.Command("clear")
+	clearCmd.Stdout = os.Stdout
+	clearCmd.Run()
+}
+
+func KeyListener(inputChan chan rune, doneChan chan bool) {
```

Comment:
This file starts to grow too quickly. Can we split each go routine into a dedicated file?

Reply:
Is it possible to make that refactoring in another PR?